src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1353,4 +1353,7 @@ I agree to all of the foregoing:</value>
   <data name="Entire_Solution" xml:space="preserve">
     <value>Entire Solution</value>
   </data>
+  <data name="CSharp_VB_Language_Server_Client" xml:space="preserve">
+    <value>C#/Visual Basic Language Server Client</value>
```

Comment:
text is fine.  but the variable name should be `CSharp_Visual_Basic_Language_Server_Client` to match it.